middle_end/flambda2/types/env/typing_env.ml

```diff
@@ -109,8 +123,8 @@ let is_empty t =
 let aliases t =
   Cached_level.aliases (One_level.just_after_level t.current_level)
 
-(* CR mshinwell: Should print name occurrence kinds *)
-(* CR mshinwell: Add option to print [aliases] *)
+(* CR-someday mshinwell: Should print name occurrence kinds *)
+(* CR-someday mshinwell: Add option to print [aliases] *)
```

Comment:
I think this comment dates from when aliases where never printed. Now that they're always printed, I'm less convinced that we need an option.